app/uk/gov/hmrc/incorporatedentityidentificationfrontend/config/AppConfig.scala

```diff
@@ -29,19 +29,9 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig, config: Configuration)
   lazy val selfBaseUrl: String = servicesConfig.baseUrl("self")
   lazy val selfUrl: String = servicesConfig.getString("microservice.services.self.url")
 
-  private lazy val contactHost: String = servicesConfig.getString("contact-frontend.host")
+  lazy val contactHost: String = servicesConfig.getString("contact-frontend.host")
 
-  private lazy val assetsUrl: String = servicesConfig.getString("assets.url")
-
-  lazy val assetsPrefix: String = assetsUrl + servicesConfig.getString("assets.version")
-  lazy val analyticsToken: String = servicesConfig.getString(s"google-analytics.token")
-  lazy val analyticsHost: String = servicesConfig.getString(s"google-analytics.host")
-
-  def reportAProblemPartialUrl(serviceIdentifier: String): String =
-    s"$contactHost/contact/problem_reports_ajax?service=$serviceIdentifier"
-
-  def reportAProblemNonJSUrl(serviceIdentifier: String): String =
-    s"$contactHost/contact/problem_reports_nonjs?service=$serviceIdentifier"
+  lazy val deskProServiceId: String = "grs"
```

Comment:
I would probably use "grsDeskProServiceId" to help differentiate from the deskProServiceId in PageConfig.

Reply:
is that a reason not to approve the pr? 😂

Reply:
I suppose not, but it would avoid potential confusion.